docs/Doxyfile.in

```diff
@@ -500,7 +487,7 @@ WARN_IF_UNDOCUMENTED   = NO
 # parameters in a documented function, or documenting parameters that 
 # don't exist or using markup commands wrongly.
 
-WARN_IF_DOC_ERROR      = YES
+WARN_IF_DOC_ERROR      = NO
```

Comment:
nack, we want this and we want to fix it

Reply:
OK